server/src/test/java/io/crate/types/JsonTypeTest.java

```diff
@@ -30,7 +30,8 @@
 
 import io.crate.metadata.CoordinatorTxnCtx;
 
-class JsonTypeTest {
+// TODO convert to DataTypeTestCase and add DataGenerator
```

Comment:
I don't think it would be too much effort to add a DataGenerator for this type but the PR is big enough already so we can do it in a follow-up.